packages/devtools-reps/src/object-inspector/utils/load-properties.js

```diff
@@ -88,6 +94,10 @@ function mergeResponses(responses: Array<Object>) : Object {
     if (response.prototype) {
       data.prototype = response.prototype;
     }
+
+    if (typeof response === "string") {
```

Comment:
This looks brittle to me, I guess we could have in the future more than 1 response being a string.
In l.76, could we put into the `promises` array something more predictable ? Something like
```js
if (shouldLoadFullText(item, loadedProperties)) {
  promises.push(
    getFullText(getObjectClient(), getValue(item))
      .then(fullString => ({fullString}))
  );
}
```

This way we can test the `fullString` property existence here.

Reply:
I didn't like the generic string check either but I haven't thought of postprocessing the promise to be honest!